user_service/README.md

```diff
@@ -80,7 +80,7 @@ Replace <access_token> with a valid access token obtained by logging in using th
 
 ### Get User by ID 
 
-- **URL**: `/api/users/<str:username>`
+- **URL**: `/user-service/getUser/<str:username>`
```

Comment:
(I'm assuming we should follow [REST resource-naming](https://restfulapi.net/resource-naming/)). Perhaps something like `/user-service/users/<str:username>` would adhere to the convention better. Same goes for the rest of the endpoints

Reply:
Updated